assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js

```diff
@@ -53,7 +58,9 @@ const CategoryPillInnerContainer = styled.div`
 // Flips the button upside down when expanded;
 // Important: the visibily is 'inherit' when props.visible because otherwise
 // it gets shown even when the provider is not the selectedProvider!
-const ExpandButton = styled(ArrowDown)`
+const ExpandButton = styled(ArrowDown).attrs({
+  'aria-label': __('Expand Categories', 'web-stories'),
```

Comment:
If you'd add the attribute below you could do:

```js
aria-label={isExpanded ? __('Collapse Categories', 'web-stories') :  __('Expand Categories', 'web-stories')}
```

Reply:
Done.